src/couch/src/couch_lockout.erl

```diff
@@ -0,0 +1,50 @@
+% Licensed under the Apache License, Version 2.0 (the "License"); you may not
+% use this file except in compliance with the License.  You may obtain a copy of
+% the License at
+%
+%   http://www.apache.org/licenses/LICENSE-2.0
+%
+% Unless required by applicable law or agreed to in writing, software
+% distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+% WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+% License for the specific language governing permissions and limitations under
+% the License.
+
+-module(couch_lockout).
+-compile(tuple_calls).
```

Comment:
Is this necessary? Wasn't it removed in OTP 21?

Reply:
It is needed for the MochiReq:get(peer) call at the end of the module.

Reply:
But maybe we can get completely rid of it? https://github.com/mochi/mochiweb/pull/215